docs/data-sources.md

```diff
@@ -110,9 +131,9 @@ zoom   | meters at equator     | arc seconds     | nominal arc degrees minutes s
 
 ## Data updates
 
-Terrain tiles were built during 2016Q2 and 2016Q3 based on available sources at that time. Regular updates are not planned.
+Terrain tiles version 1 was built during 2016Q2 and 2016Q3 based on available sources at that time. Version 2 was built during 2017Q3. Regular updates are not planned.
```

Comment:
We've been calling this v2, but in terms of our API nothing is changing so we'd probably not advertise this as v2 in our API endpoints? Depending on final implementation, this reference here may need to change.

Reply:
We're now calling this `v1.1` since it's (for the service) only an improvement of the data.

Reply:
Fixed in 6e15ef0.